src/components/cards/type1.stories.jsx

```diff
@@ -0,0 +1,287 @@
+				closable={ false }
+				className="py-0"
+			/>
+		),
+	},
+	{
+		id: '4',
+		value: 'security',
+		icon: <Shield />,
+		label: 'Security',
+		description: 'Update your security settings and passwords.',
+		toggleLabel: 'Security',
+		hideSelection: true,
+		useSwitch: false,
+		bagde: (
+			<Badge
+				label={ 'PRO' }
+				size="md"
+				icon={ null }
+				variant="inverse"
+				closable={ false }
+				className="py-0"
+			/>
+		),
+	},
+	{
+		id: '5',
+		value: 'marketing',
+		icon: <AppWindow />,
+		label: 'Marketing Button',
+		description: 'Demo',
+		toggleLabel: 'Billing',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '6',
+		value: 'modal',
+		icon: <PictureInPicture />,
+		label: 'Modal',
+		description: 'Demo',
+		toggleLabel: 'modal',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '7',
+		value: 'socialshare',
+		icon: <Share2 />,
+		label: 'Social Share',
+		description: 'Demo',
+		toggleLabel: 'socialshare',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '8',
+		value: 'Blockquote',
+		icon: <Newspaper />,
+		label: 'Blockquote',
+		description: 'Demo',
+		toggleLabel: 'blockquote',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '9',
+		value: 'contenttimeline',
+		icon: <ChartNoAxesGantt />,
+		label: 'Content Timeline',
+		description: 'Demo',
+		toggleLabel: 'contenttimeline',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '10',
+		value: 'googlemaps',
+		icon: <Map />,
+		label: 'Google Maps',
+		description: 'Demo',
+		toggleLabel: 'googlemaps',
+		hideSelection: false,
+		useSwitch: true,
+		bagde: (
+			<Badge
+				label={ 'New' }
+				size="md"
+				icon={ null }
+				variant="green"
+				closable={ false }
+				className="py-0 mr-2"
+			/>
+		),
+	},
+	{
+		id: '11',
+		value: 'lottieanimation',
+		icon: <House />,
+		label: 'Lottie Animation',
+		description: 'Demo',
+		toggleLabel: 'lottieanimation',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '12',
+		value: 'animations',
+		icon: <PanelLeftClose />,
+		label: 'Animations',
+		description: 'Demo',
+		toggleLabel: 'animations',
+		hideSelection: false,
+		useSwitch: true,
+		bagde: (
+			<Badge
+				label={ 'Extension' }
+				size="md"
+				icon={ null }
+				variant="yellow"
+				closable={ false }
+				className="py-0 mr-2"
+			/>
+		),
+	},
+];
+
+// Template for Type 1
+const Template1 = () => {
+	const radioButtonGroupData = defaultRadioButtonGroupData;
+
+	return (
+		<Container
+			containerType="flex"
+			direction="column"
+			className="md:w-full lg:w-full border border-solid rounded-md border-border-subtle p-4"
+			gap="xs"
+		>
+			<Container.Item className="md:w-full p-1 lg:w-full">
+				<Container justify="between" gap="xs">
+					<Container.Item className="">
+						<Label className="font-semibold">Blocks</Label>
+					</Container.Item>
+					<Container.Item className="items-center flex" gap="xs">
+						<Label>
+							View All <ArrowUpRight />{ ' ' }
+						</Label>
```

Comment:
This should be a Button component; please refer to the screenshot: https://bsf.d.pr/i/0PQZOG
Additionally, there should be some gap between the 'View All' button and the three-dots icon regarding mockup, as shown here: https://bsf.d.pr/i/LCX6t2